torchao/quantization/quant_api.py

```diff
@@ -401,10 +404,25 @@ def insert_subclass(lin):
     return insert_subclass
 
 def quantize_(
+    # NOT FOR LAND: this is the same across inference and torchao.float8
     model: torch.nn.Module,
+
+    # NOT FOR LAND: this seems to leak implementation details, in the future
+    # can this be named in a more generic way without mentioning the 
+    # implementation? Short term, seems ok to have float8 training use this
+    # evern though the implementation is different from 100% tensor subclass 
+    # based workflows.
     apply_tensor_subclass: Callable[[torch.nn.Module], torch.nn.Module],
```

Comment:
yeah this should be named something different like constructor or transform function.